components/mapSearchBar/index.js

```diff
@@ -216,26 +216,26 @@ class MapSearchBar extends Component {
 
       // if H- or VL- prefix entered by user only show relevant indoor predictions
       if (destination.startsWith('h-') || destination.startsWith('vl-')) {
-        const allPredictions = currentBuilding
+        const allPredictions1 = currentBuilding
           ? [currentBuilding].concat(predictions.slice(0, MAX_NUM_PREDICTIONS - 1))
           : predictions.slice(0, MAX_NUM_PREDICTIONS);
-        return allPredictions;
+        return allPredictions1;
       }
 
       if (predictions.length === 0) {
-        const allPredictions = currentBuilding ? [currentBuilding].concat(googleApiPredictions) : googleApiPredictions;
-        return allPredictions;
+        const allPredictions2 = currentBuilding ? [currentBuilding].concat(googleApiPredictions) : googleApiPredictions;
+        return allPredictions2;
```

Comment:
Did SonarQube suggest the numeric suffixes? This can confuse a developer into thinking there are three types of predictions. It is more readable if we have let allPredictions at the top then assign the content inside the if's